### ISSUE TITLE
Misspelled the Word "JavaScript" on the "WCAG 2 FAQ" Page

### ISSUE THREAD

**Issue by jfc3-dol** [w3c/wai-intro-wcag#133]
The word "**JavaScript**" on the "WCAG 2 FAQ" Page ( https://www.w3.org/WAI/standards-guidelines/wcag/faq/ ) is misspelled in two places on the page. It is found in the "Page Contents" section, and then the one where the text "[Can I meet WCAG 2 with Javascript/Ajax, Silverlight, and other technologies?](https://www.w3.org/WAI/standards-guidelines/wcag/faq/#othertechs ). The "**S**" in JavaScript should be capitalized.

**Comment by shawna-slh** [comment]
Thanks,  @jfc3-dol 

Fixed: https://github.com/w3c/wai-intro-wcag/commit/4b0b500f168d9dd62d943baa7f1086cbada41340

Will go live next time I publish the site (probably later this week).

**Comment by jfc3-dol** [comment]
@shawna-slh,

You're welcome and glad I could help.

**Comment by shawna-slh** [comment]
Can you make all my GitHub issues this easy to address!?! :-)

**Comment by jfc3-dol** [comment]
I will do what I can.

I have found other issues similar to this one or had broken or redirected links in the USWDS, 18F, Section 508 Coordinators, etc., pages on testing and development.
